api/src/main/java/org/apache/cloudstack/api/command/user/vm/DeployVMCmd.java

```diff
@@ -203,6 +203,9 @@
     @Parameter(name = ApiConstants.EXTRA_CONFIG, type = CommandType.STRING, since = "4.12", description = "an optional URL encoded string that can be passed to the virtual machine upon successful deployment", length = 5120)
     private String extraConfig;
 
+    @Parameter(name = "copyimagetagstovm", type = CommandType.BOOLEAN, since = "4.13", description = "if true we copy the image tags to the deployed Vm; defaulted to false if not specified")
```

Comment:
@anuragaw can you move/create the string in ApiConstants than hard code it here.